lib/rubycritic/core/analysed_modules_collection.rb

```diff
@@ -30,6 +30,10 @@ def to_json(*options)
       @modules.to_json(*options)
     end
 
+    def size
+      @modules.size
```

Comment:
I'd prefer using delegation via Ruby's Forwardable module here to communicate our intent clearer.